### PR TITLE
Add PhpWebStudy.app v1.0.14

### PR DESCRIPTION
**Pull request by xpf0000**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by bevanjkay** [comment]
Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

**Comment by xpf0000** [comment]
ok, i had add a zap stanza, thank's for review

File: Casks/phpwebstudy.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+cask "phpwebstudy" do
+  version "1.0.14"
+  sha256 "e7d4ef8a485e69513ac0f860be013dd3405c0bcb2d474aa3bdeba4caf4257f05"
+
+  url "https://github.com/xpf0000/PhpWebStudy/releases/download/v#{version}/PhpWebStudy-#{version}-mac.zip", verified: "github.com/xpf0000/PhpWebStudy/"
+  name "phpwebstudy"
```
Comment (bevanjkay) [inline Casks/phpwebstudy.rb:6]: Looking at the website the name should likely be this, as it should be stylised the same as in the upstream application.
```suggestion
  name "PhpWebStudy"
```

**Review by bevanjkay — DISMISSED** [review]

**Comment by miccal** [comment]
Thank you @xpf0000.

**Review by miccal — APPROVED** [review]

**Pull request merged**